src/mocks.ts

```diff
@@ -533,11 +532,9 @@ const globals = {
         tmpState.set(key, value);
       }
 
-      const val = JSON.stringify(value);
-
-      IOGas = IOGas + val.length; // Total serialized length of gas
+      IOGas = IOGas + JSON.stringify(value).length; // Total serialized length of gas
 
-      return val ?? "null"; //insta.exports.__newString(val);
+      return value;
     },
     "db.delObject": (keyPtr: string) => {
       const key = keyPtr; //(insta as any).exports.__getString(keyPtr);
```

Comment:
The following `tmpState` value should be the value `null` not the string `"null"`